### PR TITLE
README.md: partly incorporate changes from discussion on MR #782.

### PR DESCRIPTION
**Pull request by Akira25**
This implements some of the further suggestions from #782.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,11 @@ More user relevant information about the firmware are on the wiki page at: https
 * [ReleaseNotes](https://wiki.freifunk.net/Berlin:Firmware/v1.0.2)
 * a tutorial ([en](https://wiki.freifunk.net/Berlin:Firmware:En:Howto) / [de](https://wiki.freifunk.net/Berlin:Firmware:Howto)) on router configuration
 
-For questions write a mail to <berlin@berlin.freifunk.net>, open a discussion with 
-@freifunk-berlin/firmware or come to our weekly meetings.
+For questions write a mail to our public mailinglist <berlin@berlin.freifunk.net>, open a discussion with
+@freifunk-berlin/firmware or come to our weekly meetings. For futher contact information have a look at https://berlin.freifunk.net/contact/
```
Comment (SvenRoederer) [inline README.md:15]: is it really "futher"?

Probably also change line-length to 80 char, as discussed in https://github.com/freifunk-berlin/firmware/pull/825#discussion_r449887475

Comment (Akira25) [reply README.md]: Corrected.

**Comment by Akira25** [comment]
#825 needs to be merged first, as this branch is rebased on that branch.

**Comment by SvenRoederer** [comment]
When this PR depends on #825 and as both deal with the README, I suggest to have both commits in a common PR and not keeping them so separate. It's quite confusing to me and also puts additional work on you, maintaining to referencing PRs.

Probably push the 2. commit of this PR onto #825, update the PR-description and have it handled there, then close this PR. I did a similar thing in PR #807.

**Pull request closed without merging**